docs/source/data/data_catalog.md

```diff
@@ -1,31 +1,34 @@
 
-All supported data connectors are available in [`kedro-datasets`](/kedro_datasets).
+The example below registers two `csv` datasets, and an `xlsx` dataset. The minimum details needed to load and save a file within a local file system are the key, which is name of the dataset, the type of data to indicate the dataset to use (`type`) and the file's location (`filepath`).
 
-## Use the Data Catalog within Kedro configuration
-
-Kedro uses configuration to make your code reproducible when it has to reference datasets in different locations and/or in different environments.
-
-You can copy this file and reference additional locations for the same datasets. For instance, you can use the `catalog.yml` file in `conf/base/` to register the locations of datasets that would run in production, while copying and updating a second version of `catalog.yml` in `conf/local/` to register the locations of sample datasets that you are using for prototyping your data pipeline(s).
+```yaml
+companies:
+  type: pandas.CSVDataSet
+  filepath: data/01_raw/companies.csv
 
-Built-in functionality for `conf/local/` to overwrite `conf/base/` is [described in the documentation about configuration](../configuration/configuration_basics.md). This means that a dataset called `cars` could exist in the `catalog.yml` files in `conf/base/` and `conf/local/`. In code, in `src`, you would only call a dataset named `cars` and Kedro would detect which definition of `cars` dataset to use to run your pipeline - `cars` definition from `conf/local/catalog.yml` would take precedence in this case.
+reviews:
+  type: pandas.CSVDataSet
+  filepath: data/01_raw/reviews.csv
 
-The Data Catalog also works with the `credentials.yml` file in `conf/local/`, allowing you to specify usernames and passwords required to load certain datasets.
+shuttles:
+  type: pandas.ExcelDataSet
+  filepath: data/01_raw/shuttles.xlsx
+  load_args:
+    engine: openpyxl # Use modern Excel engine (the default since Kedro 0.18.0)
+```
+### Dataset `type`
 
-You can define a Data Catalog in two ways - through YAML configuration, or programmatically using an API. Both methods allow you to specify:
+Kedro offers a range of datasets, including CSV, Excel, Parquet, Feather, HDF5, JSON, Pickle, SQL Tables, SQL Queries, Spark DataFrames and more. They are supported with the APIs of pandas, spark, networkx, matplotlib, yaml and more.
 
- - Dataset name
- - Dataset type
- - Location of the dataset using `fsspec`, detailed in the next section
- - Credentials needed to access the dataset
- - Load and saving arguments
- - Whether you want a [dataset or ML model to be versioned](kedro_io.md#versioning) when you run your data pipeline
+[The `kedro-datasets` package documentation](/kedro_datasets) contains a comprehensive list of all available file types.
 
-## Specify the location of the dataset
+### Dataset `filepath`
```

Comment:
Also h3 seems to be only marginally smaller than h2? At the beginning I thought they were the same section

(Sorry because I know these are not a problem in this PR, but reading while paying close attention is making me discover these things)

Reply:
Also in the ticket.